Assets/Scripts/Augment/AugmentImplementations/AmmoBoxCollector.cs

```diff
@@ -0,0 +1,35 @@
+using System.Collections;
+using UnityEngine;
+
+public class AmmoBoxCollector : MonoBehaviour
+{
+    private const string ammoBoxBodyName = "Gatling";
```

Comment:
Should be exhanged with a serializeField refference to the corresponding Item scriptableobject and just compare  it like so: `hasAmmoBoxBody = player.identity.Body == ammoBoxBodyItem;`